ios/Sources/Repository/Authenticator.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  Authenticator.swift
+//  
+//
+//  Created by watanavex on 2021/06/12.
+//
+
+import Foundation
+import DroidKaigiMPP
+import FirebaseAuth
+
+class AuthenticatorImpl: Authenticator {
+
+    func currentUser(completionHandler: @escaping (DroidKaigiMPP.User?, Error?) -> Void) {
```

Comment:
I think this completionHandler's argument type should be `Result<DroidKaigiMPP.User, Error>` . Is it possible?

Reply:
This is an interface defined in Kotlin.
And this is the suspend function. The callback arguments are automatically mapped by the KotlinNative compiler😂

Reply:
I see. It's okey 😭